docs/integrations/web-servers/opentelemetry/squid-proxy-opentelemetry.md

```diff
@@ -0,0 +1,147 @@
+
+Path of the log file configured to capture Squid logs is needed to be given here.
+
+The files are typically located in /var/log/squid/access.log.
+You can add any custom fields which you want to tag along with the data ingested in sumo.
+Click on the **Download YAML File** button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Squid-Proxy-OpenTelemetry/Squid-Proxy-YAML.png')} alt="YAML" />
+
+## Step 3: Sending logs to Sumo
+
+Once you have the yaml file downloaded in step 2, please follow the below steps based on your environment
+
+<Tabs
+  className="unique-tabs"
+  defaultValue="Linux"
+  values={[
+    {label: 'Linux', value: 'Linux'},
+    {label: 'macOS', value: 'macOS'},
+  ]}>
+
+<TabItem value="Linux">
+
+1.  Copy the yaml at /etc/otelcol-sumo/conf.d/ folder in the Squid instance which needs to be monitored.
+2.  Restart the otelcol-sumo process using the below command 
+```sh
+sudo systemctl restart otelcol-sumo
+```
+
+</TabItem>
+<TabItem value="macOS">
+
+1.  Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Squid instance which needs to be monitored.
+2.  Restart the otelcol-sumo process using the below command 
+```sh
+ otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --conf "glob:/etc/otelcol-sumo/conf.d/*.yaml"
+```
+
+</TabItem>
+</Tabs>
+
+After successful execution of the above command, Sumo will start receiving the data from your host machine. 
+
+Press **Next**. This will install the app to your Sumo Logic Org. The app consists of Dashboards.
+
+Panels will start to fill automatically. It's important to note that each panel fills with data matching the time range query and received since the panel was created. Results won't immediately be available. Your data should appear in the dashboard within 20 minutes and you'll see full graphs and map.
+
+### Sample Log Messages in Non-Kubernetes environments
+
+```sql
+1674483805.439 44 192.168.100.40 TCP_REFRESH_MODIFIED/301 514 GET http://openstack.org/ - HIER_DIRECT/192.168.100.40 text/html
+```
+
+### Sample Query
+
+This sample Query is from the Squid Proxy - HTTP Response Analysis > URLs Experiencing Redirections panel.
+
+Query String
+
+```sql
+%"sumo.datasource"=squidproxy %"webengine.cluster.name"=*  host.name=* %"webengine.system"=squidproxy
+| json "log" as message nodrop 
+| if (isEMpty(message), _raw, message) as message
+| parse regex field = message "(?<time>[\d.]+)\s+(?<elapsed>[\d]+)\s+(?<remotehost>[^\s]+)\s+(?<action>[^/]+)/(?<status_code>[\d]+)\s+(?<bytes>[\d]+)\s+(?<method>[^\s]+)\s+(?<url>[^\s]+)\s(?<rfc931>[^\s]+)\s+(?<peerstatus>[^/]+)/(?<peerhost>[^\s]+)\s+(?<type>[^\s|$]+?)(?:\s+|$)" nodrop | parse field = message "Connection: *\\r\\n" as connection_status nodrop | parse field = message "Host: *\\r\\n" as host nodrop | parse field = message "User-Agent: *\\r\\n" as user_agent nodrop | parse field = message "TE: *\\r\\n" as te nodrop
```

Comment:
@sumoanema is there a spelling error in this line? `parse field = message "TE: *\\r\\n" as te nodrop` If that's intentional, please let me know and I'll override.

Reply:
@kimsauce - this is directly copy pasted from the panel query. Log format which we are processing through the query need this. You can override.